bin/jade.js

```diff
@@ -157,7 +158,7 @@ function watchFile(path, base) {
     if (curr.mtime.getTime() === 0) return;
     // istanbul ignore if
     if (curr.mtime.getTime() === prev.mtime.getTime()) return;
-    tryRender(base || path);
+    tryRender(base || path, rootPath);
```

Comment:
This is in `watchFile`, where `program.watch` is always true and `render === tryRender`, so I didn't bother to change it.